src/test/ui/const-generics/const_evaluatable_checked/object-safety-ok-infer-err.stderr

```diff
@@ -0,0 +1,12 @@
+error[E0284]: type annotations needed: cannot satisfy `the constant `use_dyn::<{_: usize}>::{constant#0}` can be evaluated`
```

Comment:
that is a very confusing error in so many ways. The initial message is like a three-layer error message, and the repetition on the `use_dyn` call does not actually convey new information. Afaict this is pre-existing, but should be added to the stabilization blockers for `const_evaluatable_checked`

Reply:
yeah, added that test because the error message was garbage :laughing: 

will add a fixme though